src/main/java/org/openjdk/jextract/impl/IncludeHelper.java

```diff
@@ -84,6 +84,11 @@ static IncludeKind fromScoped(Declaration.Scoped scoped) {
     private final EnumMap<IncludeKind, Set<String>> includesSymbolNamesByKind = new EnumMap<>(IncludeKind.class);
     private final Set<Declaration> usedDeclarations = new HashSet<>();
     public String dumpIncludesFile;
+    public String sharedSymbolsFile;
+
+    public String getSharedSymbolsFile() {
+        return sharedSymbolsFile;
+    }
```

Comment:
Uhm, I think this should just be in `Options`, not in `IncludeHelper`, since it's not related to the `--include` options.